components/Footer/LongFooter.tsx

```diff
@@ -78,11 +78,19 @@ export default function LongFooter({ locale = 'en' }: { locale?: string }): Reac
       ]
     },
     {
-      header: t('Foundation'),
+      header: 'Contact',
       list: [
         {
           url: 'https://makerdao.com/contact',
-          title: t('Contact')
+          title: t('Press & media')
+        },
+        {
+          url: 'https://discord.gg/2sWcgCDWCX',
+          title: 'Governance UI Discord'
+        },
+        {
+          url: 'https://chat.makerdao.com/',
```

Comment:
There's a rocket chat link in social too

Reply:
yeep , but it was more in case uses "CTRL + F" search for contact or whatever